tasks/build/versioned_links.js

```diff
@@ -12,10 +12,7 @@ module.exports = function (grunt) {
       return resolve(rootPath, file);
     });
 
-    //We don't want to build os packages with symlinks
-    let transferFiles = (source, link) => grunt.option('os-packages')
-      ? exec('cp', ['-r', source, link])
-      : exec('ln', ['-s', source, link]);
+    let transferFiles = (source, link) => exec('cp', ['-r', source, link]);
```

Comment:
I removed the symlink behavior altogether rather than updating this to rely on `skip-os-packages` because how we create our builds should be consistent regardless of whether we intend to create rpm/deb targets.